internal/dataplane/kong_client.go

```diff
@@ -461,14 +461,25 @@ func (c *KongClient) sendOutToGatewayClients(
 	ctx context.Context, s *kongstate.KongState, config sendconfig.Config,
 ) ([]string, error) {
 	gatewayClients := c.clientsProvider.GatewayClients()
-	c.logger.V(util.DebugLevel).Info("sending configuration to gateway clients", "count", len(gatewayClients))
+	previousSHAs := c.SHAs
```

Comment:
Shouldn't this only be set after the config is sent 🤔 as it was done previously in line 471?

Reply:
done.